addons/web/static/src/core/ui/ui_service.js

```diff
@@ -31,13 +33,42 @@ export function useActiveElement(refName) {
         (el) => {
             if (el) {
                 uiService.activateElement(el);
+                trapFocus(el);
                 return () => uiService.deactivateElement(el);
             }
         },
         () => [owner.el]
     );
 }
 
+function trapFocus(element) {
+    const focusableEls = getTabableElements(element);
+    const firstFocusableEl = focusableEls[0];
+    const lastFocusableEl = focusableEls[focusableEls.length - 1];
+
+    element.addEventListener("keydown", function (e) {
+        const keyPressed = getActiveHotkey(e);
```

Comment:
nitpicking: I am pretty sure that in every other place where the `getActiveHotkey` function is called, the result is stored in a constant called `hotkey`.